Automattic/wp-calypso
# Recent Feed Overhaul: Add FullPostView layout prop

Related to https://github.com/Automattic/loop/issues/194

## Proposed Changes

* This PR [introduces](https://github.com/Automattic/wp-calypso/pull/96072/files#diff-b80c57830709742a310ed2c8c14ac83cfbe27334274e4798d4fb2dbac64e1411R96) a `layout` prop on `FullPostView`.
* This allows us to customize the Full Post View for the "recent" overhaul.
* This PR adds an author/site icon, site title, and subscriber count to the Full Post View header.
* This PR removes the tags from the header and the "recommended posts" section from the bottom of the Full Post View.
* This PR is backward compatible and does not introduce changes to any current implementations of the FullPostView component.

> [!NOTE]  
> The code diff for this PR is easier to review if you enable the "Hide whitespace" option.

Before | After
--|--
<img width="1724" alt="Screenshot 2024-11-06 at 3 32 18 PM" src="https://github.com/user-attachments/assets/a7ce3b40-5c54-4cae-9aff-d75ab21f8ece">  | <img width="1725" alt="Screenshot 2024-11-06 at 3 32 30 PM" src="https://github.com/user-attachments/assets/25690488-15ce-472d-8ec3-0a4c2d745bf1">





## Why are these changes being made?


* Push the Recent Feed Overhaul project forward by making the Full Post View closer to the comps (seen in the GitHub issue).

## Testing Instructions



* Go to /read?flags=reader/recent-feed-overhaul on Calypso Live
* View that the Full Post looks good and similar to the comps found on the issue.
* View some existing implementations of the FullPostView component (such as https://wordpress.com/read/feeds/37010753/posts/5443963177) and observe that there are no regressions.

## Pre-merge Checklist
